package-lock.json

```diff
@@ -0,0 +1,865 @@
+{
```

Comment:
I'm a node noob, do all these dependencies need to be explicitly defined?

Reply:
yeah this makes the build deterministic.  If you don't include it then when npm goes to pull dependencies of dependencies then it may pull a different version if that project updated its dependencies.  this file locks all of the child dependencies and npm inc. wont allow maintainers to override these versions.  It prevents random breaks and helps against people swapping in malicious packages